install.sh

```diff
@@ -98,7 +98,10 @@ download_and_install() {
 
   # install to PNPM_HOME, defaulting to ~/.pnpm
   tmp_dir="$(mktemp -d)" || abort "Tmpdir Error!"
-  trap 'rm -rf "$tmp_dir"' EXIT INT TERM HUP
+  # Use double quotes with single-quoted variable to interpolate at trap setup time.
+  # This ensures the directory path is captured even if tmp_dir goes out of scope.
+  # shellcheck disable=SC2064
+  trap "rm -rf $tmp_dir" EXIT INT TERM HUP
```

Comment:
isn't the solution in the description this? 
```suggestion
  trap "rm -rf '$tmp_dir'" EXIT INT TERM HUP
```

Reply:
Copilot suggested changing it to trap "rm -rf $tmp_dir" EXIT INT TERM HUP once again, but I don't know why that record (or: suggestion/entry) is missing now.

Reply:
So I'm not sure which one would be better. However, I have tested both of them.

<img width="514" height="908" alt="image" src="https://github.com/user-attachments/assets/a73a4761-ad02-4697-aa3f-80ad8fc8860b" />